webaccess/res/websocket.js

```diff
@@ -29,20 +29,17 @@ function connect() {
   //alert(\"Websocket open!\");
  };
 
- websocket.onclose = function (e) {
-  console.log(
-    "QLC+ connection is closed. Reconnect will be attempted in 1 second.",
-    e.reason
-  );
+ websocket.onclose = function(ev) {
+  console.log("QLC+ connection is closed. Reconnect will be attempted in 1 second.");
   setTimeout(function () {
     connect();
   }, 1000);
-};
+ };
 
-websocket.onerror = function (err) {
-  console.error("QLC+ connection encountered error: ", err.message, "Closing socket");
+ websocket.onerror = function(ev) {
+  console.error("QLC+ connection encountered error. Closing socket");
```

Comment:
Codacy has a fix for the issue: [Insert `··`](https://app.codacy.com/gh/mcallegari/qlcplus/pullRequest?prid=9368415)
```suggestion
    console.error("QLC+ connection encountered error. Closing socket");
```